hooks/scripts/check-print-keyword.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+## Check for print keyword used for logging in the repository.
+
+count=`grep -r 'print\s*(' . | wc -l`
```

Comment:
We are checking all files in the repo? 

Sounds like a bad idea. Grep is pretty slow compared to git grep. https://stackoverflow.com/questions/17557684/is-it-better-to-use-git-grep-than-plain-grep-if-we-want-to-search-in-versioned-s


Also doesn't pre-commit pass in the files that are changed? 

Might make sense to use pythons AST module. https://stackoverflow.com/a/25249854/3888850

Reply:
Changed the way to process the print request. Closing this PR !!